modules/installation-vsphere-encrypted-vms.adoc

```diff
@@ -0,0 +1,24 @@
+// module is included in the following assemblies:
+// ../installing/installing_vsphere/installing-vsphere.adoc
+
+:_content-type: PROCEDURE
+[id="installation-vsphere-encrypted-vms_{context}"]
+= Requirements for encrypting virtual machines
+
+You can encrypt your virtual machines prior to installing {product-title} {product-version} by meeting the following requirements. 
+
+* You have configured a Standard key provider in vSphere. For more information, see link:https://docs.vmware.com/en/VMware-vSphere/7.0/com.vmware.vsphere.vsan.doc/GUID-AC06B3C3-901F-402E-B25F-1EE7809D1264.html[Adding a KMS to vCenter Server].
++
+[IMPORTANT]
+====
+The Native key provider in vCenter is not supported. For more information, see link:https://docs.vmware.com/en/VMware-vSphere/7.0/com.vmware.vsphere.security.doc/GUID-54B9FBA2-FDB1-400B-A6AE-81BF3AC9DF97.html[vSphere Native Key Provider Overview].
+====
+
+* You have enabled host encryption mode on all of the ESX-i hosts that are hosting the cluster. For more information, see link:https://docs.vmware.com/en/VMware-vSphere/7.0/com.vmware.vsphere.security.doc/GUID-A9E1F016-51B3-472F-B8DE-803F6BDB70BC.html[Enabling host encryption mode].
+* You have a vSphere account which has all cryptographic privileges enabled. For more information, see link:https://docs.vmware.com/en/VMware-vSphere/7.0/com.vmware.vsphere.security.doc/GUID-660CCB35-847F-46B3-81CA-10DDDB9D7AA9.html[Cryptographic Operations Privileges].
+
+When you deploy the OVF template in the section titled "Installing RHCOS and starting the OpenShift Container Platform bootstrap process", select the option to "Encrypt this virtual machine" when you are selecting storage for the OVF template. After completing cluster installation, create a storage class that uses the encryption storage policy you used to encrypt the virtual machines. 
```

Comment:
```suggestion
Create a storage policy that enables encryption used to encrypt the virtual machines. 
When you deploy the OVF template in the section titled "Installing RHCOS and starting the OpenShift Container Platform bootstrap process", select the option to "Encrypt this virtual machine" when you are selecting storage for the OVF template and select the newly created storage policy.
```

Reply:
> Create a storage policy that enables encryption used to encrypt the virtual machines. 
that sounds a little weird, the process

- create virtual machine storage policy with encryption enabled
- import virtual machine, click encrypt, the selection of storage devices will change based on compatibility. Select storage policy and datastore that the virtual machine will be provisioned to.

Reply:
I was going off of Hemant's statement that vSphere automatically creates a default encrypted storage policy. Isn't that separate from creating a storage *class* after deploying the VMs?

Reply:
yeah, I don't think we should be suggesting folks to create encryption storage policy at this stage.  There are bundled encryption policies with vcenter ( https://docs.vmware.com/en/VMware-vSphere/7.0/com.vmware.vsphere.security.doc/GUID-431FDB2F-7F34-468D-9D6B-BC5E95279237.html). If people want to use their own encryption policies they can, but we don't have to suggest that - at this stage.